salt/_modules/kubernetes.py

```diff
@@ -1797,3 +1797,132 @@ def replace_role(
             raise CommandExecutionError(exc)
     finally:
         _cleanup(**cfg)
+
+
+def node_annotations(name, **kwargs):
+    '''
+    Return the annotations of the node identified by the specified name
+
+    CLI Examples::
+
+        salt '*' kubernetes.node_annotations name="minikube"
+    '''
+    match = node(name, **kwargs)
+
+    if match is not None:
+        return match['metadata']['annotations']
+
+    return {}
+
+
+def node_add_annotation(node_name, annotation_name, annotation_value, **kwargs):
+    '''
+    Set the value of the annotation identified by `annotation_name` to
+    `annotation_value` on the node identified by the name `node_name`.
+    Creates the annotation if not present.
+
+    CLI Examples::
+
+        salt '*' kubernetes.node_add_annotation node_name="minikube" \
+            annotation_name="foo" annotation_value="bar"
+    '''
+    cfg = _setup_conn(**kwargs)
+    try:
+        api_instance = kubernetes.client.CoreV1Api()
+        body = {
+            'metadata': {
+                'annotations': {
+                    annotation_name: annotation_value
+                }
+            }
+        }
+        api_response = api_instance.patch_node(node_name, body)
+        return api_response
+    except (ApiException, HTTPError) as exc:
+        if isinstance(exc, ApiException) and exc.status == 404:
+            return None
+        else:
+            log.exception('Exception when calling CoreV1Api->patch_node')
+            raise CommandExecutionError(exc)
+    finally:
+        _cleanup(**cfg)
+
+    return None
```

Comment:
Is this done in the other procedures? There should be no need, either `return api_response` is reached, or the `return None` or `raise` in the `except` clause, or some other exception bubbles up. This is unreachable.

Also applies to other occurrences.

Reply:
Indeed, not sure exactly why I added them, will remove when rebasing.